src/lib/remark-runner.ts

```diff
@@ -53,7 +53,7 @@ export default function run(
           plugins: [
             'remark-preset-lint-recommended',
             ['remark-lint-list-item-indent', false],
-            ['remark-lint-ordered-list-marker-value', 'one']
+            ['remark-lint-ordered-list-marker-value', 'ordered']
```

Comment:
Should we consider removing this? It will be used when the user does not have a configuration file and is not using UI configurations